client/web/src/enterprise/site-admin/dotcom/productSubscriptions/ModelBadges.tsx

```diff
@@ -52,6 +52,8 @@ function modelBadgeVariant(model: string, mode: 'completions' | 'embeddings'): '
             case 'fireworks/starcoder':
             // Bespoke alternative models hosted for us by Fireworks. These are also allowed on the
             // Cody Gateway side
+            case 'accounts/sourcegraph/models/starcoder2-15b':
+            case 'accounts/sourcegraph/models/starcoder2-7b':
```

Comment:
```suggestion
            case 'sourcegraph/accounts/sourcegraph/models/starcoder2-15b':
            case 'sourcegraph/accounts/sourcegraph/models/starcoder2-7b':
```

but maybe we don't want to add this here yet since this is only for the UI for when we manually edit enterprise licenses which we don't need for now anyhow. And once we start needing it we probably want to use a shorthand syntax like `fireworks/starcoder2` anyways. WDYT?

Reply:
Wait actually further down you only look at `fireworks/starcoder` to get the model support so this is not needed at all